src/pages/admin/TicketCards.vue

```diff
@@ -0,0 +1,450 @@
+  },
+  data() {
+    return {
+      currentLoggedinUser: '',
+      currentLoggedinUserId: 0,
+      replyState: false,
+      TicketReply: '',
+      userToAssign: 0,
+      comments: [],
+      test: null,
+      truncateState: true,
+      dropDownState: false,
+      selectedUser: '',
+      userList: ['Arron Titus', 'Triston Lewis', 'Ross Arroyo', 'Gina'],
+    };
+  },
+  computed: {
+    assignWho() {
+      if (this.ticketData.assignee_id === 484643688) {
+        return 'Arron titus';
+      }
+      if (this.ticketData.assignee_id === 411677450351) {
+        return 'Triston Lewis';
+      }
+      if (this.ticketData.assignee_id === 114709872451) {
+        return 'Ross Arroyo';
+      }
+      if (this.ticketData.assignee_id === 403645788712) {
+        return 'Gina';
+      }
+      return this.ticketData.assignee_id;
+    },
+    assignWhoPicture() {
+      if (this.ticketData.assignee_id === 484643688) {
+        return require('@/assets/headshots/aarontitus.jpg');
+      }
+      if (this.ticketData.assignee_id === 411677450351) {
+        return require('@/assets/headshots/triston.png');
+      }
+      if (this.ticketData.assignee_id === 114709872451) {
+        return require('@/assets/headshots/rossarroyo.jpeg');
+      }
+      if (this.ticketData.assignee_id === 403645788712) {
+        return require('@/assets/headshots/ginanewby.jpg');
+      }
+      return this.ticketData.assignee_id;
+    },
+  },
+  methods: {
+    assignWhoColor(author) {
+      if (author === 484643688) {
+        return 'bg-crisiscleanup-yellow-300';
+      }
+      if (author === 411677450351) {
+        return 'bg-crisiscleanup-yellow-300';
+      }
+      if (author === 114709872451) {
+        return 'bg-crisiscleanup-yellow-300';
+      }
+      if (author === 403645788712) {
+        return 'bg-crisiscleanup-yellow-300';
+      }
+      return '';
+    },
+
+    async currentUser() {
+      this.currentLoggedinUser = `${
+        User.find(this.$store.getters['auth/userId']).first_name
+      } ${User.find(this.$store.getters['auth/userId']).last_name}`;
+    },
+    assignWhoComments(author) {
+      if (author === 484643688) {
+        return 'Arron titus';
+      }
+      if (author === 411677450351) {
+        return 'Triston Lewis';
+      }
+      if (author === 114709872451) {
+        return 'Ross Arroyo';
+      }
+      if (author === 403645788712) {
+        return 'Gina';
+      }
+      return this.ticketData.name;
+    },
+
+    async replyToTicketOpen() {
+      if (this.currentLoggedinUser === 'Arron Titus') {
+        this.currentLoggedinUserId = 484643688;
+      }
+      if (this.currentLoggedinUser === 'Triston Lewis') {
+        this.currentLoggedinUserId = 411677450351;
+      }
+      if (this.currentLoggedinUser === 'Ross Arroyo') {
+        this.currentLoggedinUserId = 114709872451;
+      }
+      if (this.currentLoggedinUser === 'Gina') {
+        this.currentLoggedinUserId = 403645788712;
+      }
```

Comment:
_Surely_ there has to be a way to fetch these from the zendesk API??

This isn't very maintainable...
Even here, you typo'd aaron's name :wink:

Reply:
Update, I dont recall the technical details of how you are interfacing with the zendesk API, but I found this:
https://developer.zendesk.com/api-reference/ticketing/users/users/#list-users

With that data, you could easily just retrieve the admin users from CCU and find the associated zendesk user ids via the email address.